connector_redmine/README.rst

```diff
@@ -0,0 +1,67 @@
+Redmine Connector
+=================
+
+Base connector module for Redmine.
+It Allows to authenticate to a Redmine instance using the REST API.
+
+It also defines a method getUser that searches the Redmine user related
+to the Odoo user. The user login must be the same in both systems.
+
+
+Installation
+============
+
+ - Install Redmine
+     Refer to http://www.redmine.org/projects/redmine/wiki/redmineinstall
+
+ - Install python-redmine
+     sudo pip install python-redmine
+
+
+Configuration
+=============
+
+- Go to Connectors -> Redmine -> Backends
+- Create a backend
+    - Location: the url of the Redmine service
+    - Key: the REST API key of your Redmine instance
+- Click on the button to test the connection
+
+You must set one Redmine service as the default one. You must check the box `Default Redmine Service`.
+By default, the timesheets of every user will be imported from that redmine service. If a user needs to
```

Comment:
I think you should not mention the TS here and stay more generic as this module is about the base.